.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+## Filing a bug report
+
+We love to receive bug reports (we're always trying to make ShellJS more
+stable). If you've found a bug, please follow these steps:
+
+ - Search for any issues that may have been created already. We often receive
+   duplicates, and cutting down on this is helpful. If someone else has already
+   reported it, please ping that issue thread.
+ - Let us know your version of NodeJS (`node -v`), your version of ShellJS (from
+   `package.json`), your OS, and any other useful information.
+ - Give an example ShellJS command to reproduce the error.
+
+## Pull requests
+
+PRs are welcome! However, we ask that you follow a few guidelines:
+
+ - Please add tests for all changes/new features.
+ - Make sure your code passes `npm test`. Please check the CI (both Appveyor and
```

Comment:
Most proposed changes are simple enough that they work on both Windows and Unix. If someone submits something that breaks on one OS, we can just mention that if they don't figure it out themselves (and are actually willing to debug that way).